### PR TITLE
sort root directories

### PR DESCRIPTION
**Pull request by ChamseddineBhd**
Signed-off-by: chamseddine <chamseddine.benhamed@ensi-uma.tn>

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -212,11 +212,20 @@ const App = () => {
         connectNotificationsUpdateConfig,
     ]);
 
+    const updateRootDirectories = () => {
+        fetchRootFolders().then((data) => {
+            let sortedData = [...data];
+            sortedData.sort(function (a, b) {
+                return a.elementName.localeCompare(b.elementName);
+            });
+            console.log(sortedData);
```
Comment (sBouzols) [inline src/components/app.js]: Do we need to keep this log here ?

File: src/components/app.js [hunk 2]
```diff
@@ -212,11 +212,20 @@ const App = () => {
         connectNotificationsUpdateConfig,
     ]);
 
+    const updateRootDirectories = () => {
+        fetchRootFolders().then((data) => {
+            let sortedData = [...data];
+            sortedData.sort(function (a, b) {
```
Comment (sBouzols) [inline src/components/app.js:218]: Why do we need a sort ? Is directory Tree component not a better place to sort those root directories ?

**Review by sBouzols — COMMENTED** [review]
not sure if app.js is the best place to add this sort

File: src/components/app.js (hunk 2 above)
Comment (ChamseddineBhd) [reply src/components/app.js:218]: we need to sort root directories to keep trees in the same order once we delete or insert a new root directory

Comment (sBouzols) [reply src/components/app.js:218]: ok

**Review by sBouzols — APPROVED** [review]
ok

**Pull request merged**